CHANGELOG.md

```diff
@@ -8,7 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0] - 2022-08-22
```

Comment:
release not out yet. date should be changed before publish.
better to keep changes in the unreleased section until actually releasing. easier to maintain document in parallell PRs etc.